invenio_app_ils/config.py

```diff
@@ -50,11 +50,8 @@
 from invenio_circulation.transitions.transitions import (  # isort:skip
     ToItemOnLoan,
     CreatedToPending,
-    ItemInTransitHouseToItemReturned,
-    ItemOnLoanToItemInTransitHouse,
     ItemOnLoanToItemReturned,
-    PendingToItemAtDesk,
-    PendingToItemInTransitPickup,
+    ItemOnLoanToItemOnLoan,
```

Comment:
I understand the state-to-state naming, but I think `ExtendItemOnLoan` would suffice, what happens internally with the states shouldn't bother the names.

Reply:
but this one is on circulation side...create a ticket about that so we can handle it :)